build.gradle

```diff
@@ -23,7 +23,7 @@ plugins {
 }
 
 ext.ballerinaLangVersion = project.ballerinaLangVersion
-ext.puppycrawlCheckstyleVersion = "8.18"
+ext.puppycrawlCheckstyleVersion = project.puppycrawlCheckstyleVersion
 ext.testngVersion = "6.14.3"
 ext.slf4jVersion = "1.7.30"
```

Comment:
If we can move these two to the `gradle.properties` file as well, we can remove all these lines with `ext.<name>`. They'll be picked up from the gradle.properties file.

Reply:
Fixed with https://github.com/ballerina-platform/module-ballerina-reflect/pull/47/commits/2a892ab9d7e79471e0bf777eb282a8324637f3ea